### ISSUE TITLE
command-line tool not working

### ISSUE THREAD

**Issue by skolachi** [tatuylonen/wiktextract#37]
I installed wiktextract using pip3 and tried the following from the README- 
wiktwords --all --all-languages --out data.json enwiktionary-20201201-pages-articles.xml.bz2

It gave me exceptions with the following recurrent error- 
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.6/dist-packages/wikitextprocessor/lua//sandbox.lua'

**Comment by michaelshekasta** [comment]
I also get this error.

**Comment by tatuylonen** [comment]
I note that there is a double slash in the path.  Are you running on Windows, Linux, or something else?  I've only tested on Linux.

**Comment by tatuylonen** [comment]
Also, are you using a virtualenv or not?

**Comment by skolachi** [comment]
> I note that there is a double slash in the path. Are you running on Windows, Linux, or something else? I've only tested on Linux.

I ran it on Linux inside a docker container. I was not using a virtual environment.

**Comment by michaelshekasta** [comment]
Hey,

@tatuylonen Thanks for your comment and great work. Unfortunately, I also got a similar error (I'm running on macOS).

I downloaded the file from [this link](https://dumps.wikimedia.org/enwiktionary/20201101/)

After that, I am running this command:
`wiktwords --all --all-languages --out data.json enwiktionary-20201101-pages-articles.xml.bz2
`


And got this error:
```
  Traceback (most recent call last):
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/core.py", line 53, in phase2_page_handler
    ret = _global_page_handler(model, title, data)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wiktextract/wiktionary.py", line 176, in page_cb
    return page_handler(ctx, model, title, text, capture_cb, config_kwargs)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wiktextract/wiktionary.py", line 147, in page_handler
    ret = parse_page(ctx, title, text, config1)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wiktextract/page.py", line 2121, in parse_page
    datas = parse_language(ctx, config, langnode, lang, lang_code)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wiktextract/page.py", line 1985, in parse_language
    process_children(langnode)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wiktextract/page.py", line 1979, in process_children
    process_children(node)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wiktextract/page.py", line 1893, in process_children
    clean_node(config, ctx, etym_data, node,
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wiktextract/page.py", line 2245, in clean_node
    v = recurse(value)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wiktextract/page.py", line 2240, in recurse
    ret = ctx.node_to_html(value, template_fn=template_fn)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/core.py", line 1349, in node_to_html
    return to_html(self, node, template_fn=template_fn,
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/node_expand.py", line 148, in to_html
    expanded = ctx.expand(text, template_fn=template_fn,
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/core.py", line 1154, in expand
    expanded = expand_recurse(encoded, parent, templates_to_expand)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/core.py", line 1115, in expand_recurse
    t = expand_recurse(encoded_body, new_parent,
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/core.py", line 989, in expand_recurse
    ret = expand_parserfn(fn_name,
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/core.py", line 930, in expand_parserfn
    ret = invoke_fn(args, expander, parent)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/core.py", line 842, in invoke_fn
    ret = call_lua_sandbox(self, invoke_args, expander, parent)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/luaexec.py", line 292, in call_lua_sandbox
    initialize_lua(ctx)
  File "/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/luaexec.py", line 242, in initialize_lua
    lua_sandbox = open(lua_dir + "/sandbox.lua").read()
FileNotFoundError: [Errno 2] No such file or directory: '/Users/User/PycharmProjects/simple_project/venv/lib/python3.8/site-packages/wikitextprocessor/lua//sandbox.lua'
```

**Comment by tatuylonen** [comment]
I was unable to reproduce (I tried running an ubuntu container under docker, then installed python3-pip, and installed wikitextprocessor and wiktextract using pip)̣.  However, the error seems to result from lua_dir variable in wikitextprocessor/luaexec.py having a slash at the end another slash is added later when it is used (in two places in the same file).  I've committed a possible fix in the main branch in https://github.com/tatuylonen/wikitextprocessor that eliminates the double slash (note that the issue seems to be in wikitextprocessor rather than wiktextract).

**Comment by tatuylonen** [comment]
I now managed to reproduce the issue on the ubuntu container on Linux. I'm still validating whether my patch works.  It has to run for a while before the bug manifests.  The patch fixed the issue in my test.

**Comment by fpw** [comment]
I ran into the same problem. The root cause was that the pip-installed version of wikitextprocessor didn't contain the directory with the lua scripts. I manually installed it from git and then it worked. The double slash didn't matter.

**Comment by tatuylonen** [comment]
I just released a new version on pypi that should contain my fix from three weeks ago.  Could you check if the problem still exists with the new version (wikitextprocessor 0.4.92)?  (I also released a new version of wiktextract, so best to update both packages before testing)

**Comment by fpw** [comment]
The lua directory is still absent in 0.4.92:

```
$ pip3 show wikitextprocessor
Name: wikitextprocessor
Version: 0.4.92
```

```
ls ~/.local/lib/python3.8/site-packages/wikitextprocessor
__init__.py  common.py  dumpparser.py  luaexec.py      parser.py     wikihtml.py
__pycache__  core.py    languages.py   node_expand.py  parserfns.py
```

When executing `python3 setup.py sdist bdist_wheel`, you can see that the generated tar.gz doesn't contain the lua scripts.

**Comment by tatuylonen** [comment]
This should now be fixed in wikitextprocessor 0.4.93.  I'm suspecting the package_data option in setup.py behaved differently on different setuptools (or some other underlying package version), and it broke again when I generated the package on a different machine, with an older setuptools.  Anyway, as far as I can tell it should now work.

**Comment by fpw** [comment]
Can confirm it's fixed for me now, thanks!
